llvm/utils/TableGen/SearchableTableEmitter.cpp

```diff
@@ -207,12 +213,17 @@ class SearchableTableEmitter {
 
 // For search indices that consists of a single field whose numeric value is
 // known, return that numeric value.
-static int64_t getNumericKey(const SearchIndex &Index, Record *Rec) {
+static int64_t getNumericKey(const SearchIndex &Index, Record *Rec,
+                             InstrEnumMapT &InstrEnumMap) {
```

Comment:
A style nit(just mine). What about making `getNumericKey` a class function like `getIntrinsic`, so that we don't need to pass `InstrEnumMap`?

Reply:
Done. Although I removed the `InstrEnumMap` parameter, it is still useful to do this since we need to invoke the new query function `Target->getInstrIntValue`.